client/tx/factory.go

```diff
@@ -515,6 +515,9 @@ func (f Factory) getSimSignatureData(pk cryptotypes.PubKey) signing.SignatureDat
 // A new Factory with the updated fields will be returned.
 // Note: When in offline mode, the Prepare does nothing and returns the original factory.
 func (f Factory) Prepare(clientCtx client.Context) (Factory, error) {
+	if f.sequence > 0 && f.unordered {
```

Comment:
can we have a basic system test that shows that you won't be able to submit this from CLI?

Reply:
[add a test for ensuring sequence cannot be set with unordered](https://github.com/cosmos/cosmos-sdk/pull/24581/commits/cddb4bd2b7a5ba95d0a11d9023b629e5f3bf2a14)